public/data/victoriacheng15.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "Victoria Cheng",
+  "type": "personal",
+  "bio": "Full Stack Developer",
+  "avatar": "https://github.com/victoriacheng15.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/victoriacheng15",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/victoriacheng15/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/viktoriacheng15",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Hashnode",
+      "url": "https://victoriacheng15.hashnode.dev/",
+      "icon": "hashnode"
+    },
+    {
+      "name": "Follow me on Polywork",
+      "url": "https://www.polywork.com/victoriacheng15",
+      "icon": "polywork"
+    }
+  ]
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

---

Hey 👋 The bot will fix this 😄